db/post_migrate/20210207232853_fix_topic_timer_duration_minutes.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 class FixTopicTimerDurationMinutes < ActiveRecord::Migration[6.0]
   def up
+    DB.exec("DELETE FROM topic_timers WHERE status_type = 7 AND duration > #{20.years.to_i / 60 / 60 / 24}")
+    DB.exec("DELETE FROM topic_timers WHERE status_type != 7 AND duration > #{20.years.to_i / 60 / 60}")
```

Comment:
Let's avoid using any Rails helper and divisions in the migrations. How about something like

```suggestion
    DB.exec("DELETE FROM topic_timers WHERE status_type = 7 AND duration > #{20 * 365}")
    DB.exec("DELETE FROM topic_timers WHERE status_type != 7 AND duration > #{365 * 20 * 24}")
```

Reply:
I'll just use plain SQL and remove Ruby string interpolation.